libs/core/src/lib/timeline/components/timeline-node/timeline-node.component.html

```diff
@@ -0,0 +1,10 @@
+<div class="fd-timeline__node-container">
+    <div *ngIf="!glyph" class="fd-timeline__connecting-line"></div>
+    <div class="fd-timeline__node" [class.fd-timeline__node--icon]="!!glyph">
```

Comment:
maybe we need to use `*ngIf="glyph"` here?

Reply:
No, for all cases we show this element